compiler/rustc_trait_selection/src/traits/select/candidate_assembly.rs

```diff
@@ -307,17 +307,6 @@ impl<'cx, 'tcx> SelectionContext<'cx, 'tcx> {
                 self.assemble_builtin_bound_candidates(sized_conditions, &mut candidates);
             } else if lang_items.unsize_trait() == Some(def_id) {
                 self.assemble_candidates_for_unsizing(obligation, &mut candidates);
-            } else if lang_items.drop_trait() == Some(def_id)
-                && obligation.predicate.is_const_if_const()
-            {
-                // holds to make it easier to transition
-                // FIXME(fee1-dead): add a note for selection error of `~const Drop`
-                // when beta is bumped
-                // FIXME: remove this when beta is bumped
```

Comment:
@fee1-dead I'm unsure if I interpreted the comment correctly. Also, you have a FIXME for this.

Reply:
Thank you for reminding me about this, yes your interpretation is correct. I will prepare a PR to address that FIXME after this gets merged.